inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -13,6 +13,12 @@ namespace vpu {
 
 namespace {
 
+bool isOneSliceEnough(int maxBoxesNum) {
+    // boxes threshold to use only one slice
+    constexpr int boxesThreshold = 3400;
```

Comment:
Was 3650 too rough an estimate?

Reply:
one slice is not enough to allocate memory with boxes number 3650